code/game/world.dm

```diff
@@ -340,6 +340,9 @@ GLOBAL_VAR(restart_counter)
 	if (features)
 		s += ": [jointext(features, ", ")]"
 
+	s += "<br>Round time: <b>[gameTimestamp("hh:mm")]</b>"
+	s += "<br>Alert level: <b>[capitalize(get_security_level())]</b>"
```

Comment:
I get the round time, I'd like a bit more reasoning around the alert level

Reply:
To be honest there's no reason except that Skyrat/Paradise/SPLURT/Citadel/Russian servers did it and I think it's somewhat useful
I can remove it if maints prefer it gone @LemonInTheDark

Reply:
added reasoning in PR OP